src/components/UseOfFunds2024/styles/swatches/color_dark.scss

```diff
@@ -0,0 +1,43 @@
+@use 'sass:color';
+
+$white: #ffffff;
+$black: #000000;
+
+$gray100: #f5f5f5;
+$gray400: #979797;
+$gray500: #555555;
+$gray800: #323232;
+
+$blue300: #88c7f6;
+$blue300-hover: color.adjust( $blue300, $lightness: 5% );
+$blue500: #2c9bf0;
+$blue500-hover: color.adjust( $blue500, $lightness: -5% );
+
+@mixin swatch() {
+	--background: #{$gray800};
+	--text-color: #{$white};
+	--link-color: #{$blue300};
+	--link-color-hover: #{$blue300-hover};
+
+	--close-button-background: #{$gray800};
+	--close-icon-stroke: #{$white};
+
+	--usage-1-background: #10826b;
+	--usage-1-color: #{$white};
+	--usage-2-background: #9d0000;
+	--usage-2-color: #{$white};
+	--usage-3-background: #154985;
+	--usage-3-color: #{$white};
+	--usage-4-background: #{$gray500};
+	--usage-4-color: #{$white};
+
+	--benefits-list-icon: #{$white};
+	--benefits-background: #{$gray500};
+
+	--budget-line-background: #cecece;
```

Comment:
I understood for consistency we would want to pull those 4 colors up to the beginning of the file and give them a name, right?